emeis/core/tests/snapshots/snap_test_views.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# snapshottest: v1 - https://goo.gl/zC4yUc
+from __future__ import unicode_literals
+
+from snapshottest import Snapshot
+
+
+snapshots = Snapshot()
+
+snapshots["test_user_export 1"] = [
+    ["Name", "First Name", "Email", "Roles and organizations"],
+    [
+        "Marsh",
+        "Heather",
+        "dmiller@example.org",
+        """Barbara Gomez: Cody West
+Katherine Gardner: Kevin Wood""",
+    ],
+    ["Ortega", "Nicole", "vgarcia@example.com", "Michelle Harris: Andrea Cunningham"],
```

Comment:
This is currently just showing the scope name, but we'd actually prefer the full scope path (`scope.full_name()`). This fix is also still due in the emeis frontend.